docs/internal/result_handlers.md

```diff
@@ -0,0 +1,307 @@
+                table_name,
+                self._client.con,
+                schema=schema_name,
+                if_exists="append",
+                index=False,
+                chunksize=1000,
+            )
+```
+
+Command:
+```console
+time data-validation -ll DEBUG validate row -sc=oravol -tc=pg \
+-tbls=dvt_test.tab_vol_1m -pk=id --hash='*' \
+--result-handler=pg_rh.pso_data_validator_results.results \
+2>/tmp/debug.log
+
+real	2m12.332s
+user	0m53.855s
+sys	0m3.094s
+```
+
+DVT stages:
+```
+04/03/2025 12:13:09 PM-DEBUG: Build config elapsed: 0.17s
+04/03/2025 12:13:13 PM-DEBUG: Target query elapsed: 4.4s
+04/03/2025 12:13:16 PM-DEBUG: Source query elapsed: 7.68s
+04/03/2025 12:13:32 PM-DEBUG: Generate report elapsed: 15.9s
+04/03/2025 12:15:19 PM-DEBUG: Write results to PostgreSQL elapsed: 106.13s
+```
+
+`vmstat` output:
+```
+procs -----------memory---------- ---swap-- -----io---- -system-- ------cpu-----
+ r  b   swpd   free   buff  cache   si   so    bi    bo   in   cs us sy id wa st
+ 0  0      0 31727084  22580 913916    0    0    13    17  337  469  8  1 91  0  0
+ 1  0      0 31136628  22596 913916    0    0     0    12 4592 5750 14  3 83  0  0
+ 1  0      0 30500128  22604 913916    0    0     0     1 1291 1798 21  4 75  0  0
+ 1  0      0 30169196  22604 913916    0    0     0    10 1240 1754 22  1 77  0  0
+ 0  0      0 30170860  22620 913916    0    0     0     5 1169 1750  8  0 92  0  0
+ 1  0      0 30172588  22620 913916    0    0     0     0 1160 1748  7  0 92  0  0
+ 0  0      0 30173500  22628 913908    0    0     0     2 1176 1750  8  0 92  0  0
+ 0  0      0 30175160  22628 913916    0    0     0     4 1147 1726  7  0 93  0  0
+ 0  0      0 30177188  22628 913916    0    0     0     0 1060 1563  6  0 94  0  0
+ 0  0      0 30177028  22628 913916    0    0     0    14 1115 1555  8  0 92  0  0
+ 0  0      0 30176932  22628 913916    0    0     0     0 1101 1551  8  0 92  0  0
+ 0  0      0 30176932  22628 913916    0    0     0     0 1088 1545  7  0 92  0  0
+ 1  0      0 30177024  22628 913916    0    0     0     0 1109 1545  8  0 92  0  0
+ 0  0      0 30176016  22628 913916    0    0     0     0 1114 1553  8  0 92  0  0
+ 0  0      0 31735332  22636 913916    0    0     0     1 1059 1721  4  0 96  0  0
+ 0  0      0 31735292  22636 913916    0    0     0     0  954 1722  0  0 100  0  0
+```
+
+
+### PostgreSQL - Dataframe.to_sql(method=multi)
+
+Writing to PostgreSQL using the pandas `to_sql(method=multi)` method.
+```python
+            result_df.to_sql(
+                table_name,
+                self._client.con,
+                schema=schema_name,
+                if_exists="append",
+                index=False,
+                chunksize=1000,
+                method="multi",
+            )
+```
+
+Command:
+```console
+time data-validation -ll DEBUG validate row -sc=oravol -tc=pg \
+-tbls=dvt_test.tab_vol_1m -pk=id --hash='*' \
+--result-handler=pg_rh.pso_data_validator_results.results \
```

Comment:
```suggestion
--result-handler=<GCP_PROJECT_ID>.pso_data_validator_results.results \
```

Reply:
`rg_rh` is the name of a connection file, not a project name. I've updated the doc.